master-firmware/protobuf/strategy.proto

```diff
@@ -11,18 +11,18 @@ enum PuckColor {
     RED = 0;
     GREEN = 1;
     BLUE = 2;
-    WHITE = 3;
+    REDGREEN = 3;
```

Comment:
nitpick: add comment specifying that it means red or green, aka unknown ?

Reply:
what if I rename it `RED_OR_GREEN`?